src/JsonLd/Serializer/ItemNormalizer.php

```diff
@@ -47,6 +49,29 @@ final class ItemNormalizer extends AbstractItemNormalizer
     use JsonLdContextTrait;
 
     public const FORMAT = 'jsonld';
+    public const JSONLD_KEYWORDS = [
```

Comment:
This could maybe be an `@internal` enum?

Reply:
too much trouble transforming an enum to an array this is more efficient imo, I could set `private` visibility?